components/nimbus/ios/Nimbus/NimbusBuilder.swift

```diff
@@ -177,6 +190,15 @@ public class NimbusBuilder {
                 }
             }
 
+            if let callback = onFetchCallback {
+                NotificationCenter.default.addObserver(forName: .nimbusExperimentsFetched,
+                                                       object: nil,
+                                                       queue: nil)
+                { _ in
+                    callback(nimbus)
+                }
+            }
```

Comment:
I'm not familiar with the notification center piece here, can you elaborate on what that's doing?